docs/excel/custom-functions-overview.md

```diff
@@ -342,11 +342,11 @@ function secondHighest(values){
 
 ## Discovering cells that invoke custom functions
 
-Custom funtions also allows you to format ranges, display cached values, and reconcile values using `caller.address`, which makes it possible to discover the cell that invoked a custom function. You might use `caller.address` in some of the following scenarios:
+Custom funtions also allows you to format ranges, display cached values, and reconcile values using the address of the cell which invokes a custom function. You might use this functionality in some of the following scenarios:
 
-- Formatting ranges: Use `caller.address` as the key of the cell to store information in [AsyncStorage](https://docs.microsoft.com/office/dev/add-ins/excel/custom-functions-runtime#storing-and-accessing-data). Then, use [onCalculated](https://docs.microsoft.com/javascript/api/excel/excel.worksheet#oncalculated) in Excel to load the key from `AsyncStorage`.
+- Formatting ranges: Use the cell's address as the key to store information in [AsyncStorage](https://docs.microsoft.com/office/dev/add-ins/excel/custom-functions-runtime#storing-and-accessing-data). Then, use [onCalculated](https://docs.microsoft.com/javascript/api/excel/excel.worksheet#oncalculated) in Excel to load the key from `AsyncStorage`.
 - Displaying cached values: If your function is used offline, display stored cached values from `AsyncStorage` using `onCalculated`.
-- Reconciliation: Use `caller.address` to discover an origin cell to help you reconcile where processing is occurring.
+- Reconciliation: Use the cell's address to discover an origin cell to help you reconcile where processing is occurring.
```

Comment:
what's an origin cell? Is this explained anywhere?

Reply:
```suggestion
- Reconciliation: Use the cell's address to help reconcile where processing is occurring.
```
Maybe this is more clear? Basically my understanding of the large picture is that you can use the location to double check that what you think is happening in a cell is really happening there. So the "origin cell" (aka cell that invokes the function) should be the same as the cell's address, which I guess is what you would be reconciling.

Reply:
I recommend changing the heading to "Get the address of the cell that invoked your custom function" then in first sentence "When a cell value contains your custom function and it calculates the result, it invokes your custom function method. In some scenarios you'll need to track the address of the cell that invoked your custom function..." then on to the 3 scenarios. Does that work? 

And yes I like your rewrite to avoid "origin cell" otherwise I feel like we need more definition around it.

Reply:
I went ahead and updated text based on a variation of my suggestion.